### PR TITLE
[13.0][MIG] sale_last_price_info

### PR DESCRIPTION
**Pull request by Khalid-SerpentCS**
This module adds the last sale info of the product.
*  Last Sale Price
*  Last Sale Date
*  Last Customer

### CODE REVIEW

File: sale_last_price_info/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+{
+    "name": "Product Last Price Info - Sale",
+    "version": "13.0.2.0.0",
+    "author": "OdooMRP team, "
```
Comment (pedrobaeza) [inline sale_last_price_info/__manifest__.py]: Please change this to `AvanzOSC, Tecnativa`, which are the companies that are now

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by marylla — APPROVED** [review]
Hi there,
I made a functional review. LGTM! :)

**Comment by rousseldenis** [comment]
Comments attended. Merging
/ocabot merge

**Pull request merged**